TicTacToeAppTests/ViewControllerSpec.swift

```diff
@@ -1,15 +1,51 @@
+
 import Quick
 import Nimble
 import TicTacToeApp
 
 class ViewControllerSpec: QuickSpec {
+
+  override func setUp() {
+    continueAfterFailure = false
+  }
+
   override func spec() {
+    
+    var controller: ViewController!
+    var infoLabel: UILabel!
+    
+    func tagButtonsInSequence(buttons: [UIButton]) {
+      var tag = 0
+        
+      for button in buttons {
+          button.tag = tag
+          tag += 1
+      }
+    }
+    
+    func makeMovesInSequence(controller: ViewController, buttonSequence: [Int]) {
+      for buttonIndex in buttonSequence {
+          controller.makeMove(controller.boardButtons[buttonIndex])
+      }
+    }
+
+    beforeEach {
+      controller = ViewController()
```

Comment:
I would be wary of initializing the object under test in a "global" `beforeEach`. It can lock you into a given setup/state that you may not want. In this case, since you are passing nothing to the initializer, and are reasonably confident that you will not be in the future, this seems safe to do.

Various alternatives (that are not mutually exclusive):

1. Build the object in each test.
2. Have multiple `describe` blocks, each with their own `beforeEach`.
3. Have a helper function that builds the object under test for you.

These strategies can often be used in tandem with each other. This also definitely relates to how to organize tests.